### ISSUE TITLE
Confirm growth for the csc031-gem model

### ISSUE THREAD

**Issue by djinnome** [pnnl-predictive-phenomics/csc031-gem#2]

**Comment by djinnome** [comment]
The model grows using cobra, but we have yet to confirm growth in memote.

**Comment by djinnome** [comment]
In memote, we have the following report.

https://pnnl-predictive-phenomics.github.io/csc031-gem/diff5788e60b72070a4f7f6a353a1364c693c036bd0f


Even though we have an accuracy of 63%, it is attained without any true positives or false positives:  the genome-scale model for csc031 can't grow under any condition

```python
{"model.xml":{"TP":[],
"TN":[3,6,8,9,11,16,18,19,20,21,23,24,26,27,28,29,31,32,33,34,35,36,37,38,39,40,41,42,43,44,47,48,49,53,56,57,58],
"FP":[],
"FN":[0,1,2,4,5,7,10,12,13,14,15,17,22,25,30,45,46,50,51,52,54,55],
"TPR":0,"TNR":1,"PPV":null,"FDR":null,
"ACC":0.6271186440677966,"MCC":null}}
```

**Comment by mcnaughtonadm** [comment]
Good catch, this is something happening for I believe all the models. We are getting only True/False Negatives and no True/False Positives.

**Comment by JamesPino** [comment]
You can check the media. This happened when using some m9 media for one of
the other models. Turns out it was using default media, not the one
supplied in experimental.yml. There's also a growth rate you can set in
that file that if you set too high all fails.  If you don't set one, it
runs the model and uses a threshold to that growth value.

**Comment by djinnome** [comment]
Current status:  determine which base nutrient media the memote diff is running.
